src/arrayMethodSort.js

```diff
@@ -5,7 +5,35 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let count = 0;
+
+    const strCompare = (a, b) => {
+      const stringA = String(a);
+      const stringB = String(b);
+
+      return stringA > stringB
+        ? 1
+        : stringA === stringB
+          ? 0
+          : -1;
```

Comment:
```suggestion
        ? 1
        :  0
```
Нам, по суті, не важливо, чи там буде 0 чи -1. Логіка залишиться однакова у обох випадках

Reply:
Напевно.